scripts/Package_BGA/bga_xilinix.yml

```diff
@@ -0,0 +1,148 @@
+# Artix-7 FPGA Packages
+# Notes:
+#  - Often names are similiar, e.g. FB, FBG and FBV. I choose FBG as it is the common one.
+#    The addition stems from different solder types for the BGA balls
+#  - Xilinx_CPG236 and Xilinx_CPG238 are the same, different name comes from bank differences
+#  - Xilinx_CSG324 and Xilinx_CSG325 are the same, different name comes from bank differences
+#  - Xilinx_RS484 and Xilinx_SBG484 have same landing pattern but different heights/3D packages
+#  - Xilinx_RB484, Xilinx_FBG484 and Xilinx_FGG484 have same landing pattern but different heights/3D packages
+#  - Xilinx_RB676, Xilinx_FBG676 and Xilinx_FGG676 have same landing pattern but different heights/3D packages
+
+Xilinx_CPG236:
+  description: "Artix-7 BGA, 19x19 grid, 10x10mm package, 0.5mm pitch; https://www.xilinx.com/support/documentation/user_guides/ug475_7Series_Pkg_Pinout.pdf, NSMD pad definition Appendix A"
+  additional_tags: "CP236 CPG236"
+  pkg_width: 10
+  pkg_height: 10
+  pitch: 0.5
+  pad_diameter: 0.275
+  mask_margin: 0.05
+  layout_x: 19
+  layout_y: 19
+  row_skips: [[], [], [], [[4,17]], [[4,17]], [[4,17]], [[4,7],[14,17]], [[4,7],[14,17]], [[4,7],[14,17]], [[4,7],[9,12],[14,17]], [[4,7],[14,17]], [[4,7],[14,17]], [[4,7],[14,17]], [[4,17]], [[4,17]], [[4,17]], [], [], []]
+Xilinx_CPG238:
```

Comment:
Footprints with only a single package matching them don't use the `additional_tags` key; should all entries here use that for consistency even if there's only a single package mentioned?

Reply:
I think you are right, I will include available options into the tags.
Therefore it is consistent over the hole range.